/*
 * This file was automatically generated by EvoSuite
 * Fri Apr 21 19:54:19 KST 2017
 */

package com.sleepycat.je.cleaner;

import static org.evosuite.runtime.EvoAssertions.assertThrownBy;
import static org.junit.Assert.fail;

import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.Test;
import org.junit.runner.RunWith;

import com.sleepycat.je.Database;

@RunWith(EvoRunner.class) @EvoRunnerParameters(useVNET = true, separateClassLoader = true, useJEE = true)
public class VerifyUtils_ESTest7 extends VerifyUtils_ESTest_scaffolding7 {

  @Test
  public void test0()  throws Throwable  {
      // Undeclared exception!
      try { 
        VerifyUtils.checkLsns((Database) null);
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         assertThrownBy("com.sleepycat.je.DbInternal", e);
      }
  }

  @Test
  public void test1()  throws Throwable  {
      VerifyUtils verifyUtils0 = new VerifyUtils();
  }
}
